### DISCUSSION TITLE
Cannot find TSFreshFeatureExtractor

### DISCUSSION THREAD

**Discussion by lizard1234** [alan-turing-institute/sktime#1515] [Q&A]
Hello everyone,

can someone tell me how to import TSFreshFeatureExtractor?
In the tutorial it says from `sktime.transformations.panel.tsfresh import TSFreshFeatureExtractor` but it seems to be moved to another package.

Thanks for answering.

**Comment by fkiraly** [comment]
To me it looks like the right import, and I just looked and it's still there.

Can you post version numbers of packages, and the error message you get?

**Reply by lizard1234** [reply]
Thank you for your hint with the version number.
Version installed was 0.1.0 due to an old python version. I upgraded python and sktime and it works now.
Thanks :)

**Reply by fkiraly** [reply]
0.1.0.? Ah, the good old times 😄 where ts was not fresh yet...

**Reply by lizard1234** [reply]
Correct. I was working with python 3.6.x so sktime was automatically installed in version 0.1.0, therefore I couldn't import the module.
Now I'm facing another problem with tsfresh, but I'll open an other discussion for that. :)
